README.md

```diff
@@ -49,6 +49,14 @@ planned high level features:
 We've also documented the [high level design](/doc/design/high-level.md) of the
 rules.
 
+## Projects using rules_xcodeproj
+
+- [Envoy Mobile](https://github.com/envoyproxy/envoy-mobile)
+- [SwiftLint](https://github.com/realm/SwiftLint)
```

Comment:
Private repos won't have links. Which will look better: a list like this, with some of them being links, or a table with the link as a column, where the private repos don't have anything there? I thought the list way would look best.